spec/models/qing_group_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+describe QingGroup do
+  context "One QingGroup" do
+    before do
+      @form = FactoryGirl.create(:sample_form, question_types: [['text', 'text', 'text']])
+      group = create(:qing_group, form: @form, ancestry: @form.root_group.id)
```

Comment:
This should not be necessary I think. You can use the form factory directly to create a form with several groups.